### PR TITLE
[Tests-Only] used skeleton files more wisely on capabilites and comments part of API tests

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- skeleton file/folders are used more wisely on the following features suite
  - apiComments
  - apiCapabilities

## Related Issue

- Part of https://github.com/owncloud/QA/issues/658

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiComments/deleteComments.feature [hunk 1]
```diff
@@ -6,98 +6,91 @@ Feature: Comments
...
-    Given user "Alice" has created folder "/FOLDER_TO_COMMENT"
-    And user "Alice" has shared folder "/FOLDER_TO_COMMENT" with user "Brian"
-    And user "Alice" has commented with content "Folder owner comment" on folder "/FOLDER_TO_COMMENT"
-    And user "Brian" has commented with content "Sharee comment" on folder "/FOLDER_TO_COMMENT"
-    And user "Brian" should have the following comments on folder "/FOLDER_TO_COMMENT"
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has shared folder "/FOLDER" with user "Brian"
+    And user "Alice" has commented with content "Folder owner comment" on folder "/FOLDER"
+    And user "Brian" has commented with content "Sharee comment" on folder "/FOLDER"
+    And user "Brian" should have the following comments on folder "/FOLDER"
```
Comment (Talank) [inline tests/acceptance/features/apiComments/deleteComments.feature:66]: This step sounds like a `Then` step. Maybe, you should consider rewording this as well :thinking: 
Other than this, everything LGTM :+1:

File: tests/acceptance/features/apiComments/comments.feature [hunk 2]
```diff
@@ -7,12 +7,11 @@ Feature: Comments
 
   @smokeTest
   Scenario: Getting info of comments using files endpoint
-    Given user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/myFileToComment.txt"
```
Comment (individual-it) [inline tests/acceptance/features/apiComments/comments.feature:10]: what about leaving this step and for that don't use any skeleton in the Background?

File: tests/acceptance/features/apiComments/deleteComments.feature [hunk 3]
```diff
@@ -6,98 +6,91 @@ Feature: Comments
     And these users have been created with default attributes and small skeleton files:
       | username |
       | Alice    |
-      | Brian    |
 
   @smokeTest
   Scenario Outline: Deleting my own comments on a file belonging to myself
-    Given user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/myFileToComment.txt"
```
Comment (individual-it) [inline tests/acceptance/features/apiComments/deleteComments.feature:13]: same comment as for comments.feature

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiComments/deleteComments.feature (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiComments/deleteComments.feature:66]: yeah..we do not need to use then verification for given steps.
removed this! TY :clinking_glasses:

**Comment by kiranparajuli589** [comment]
@individual-it @Talank addressed your comments. please review again!

File: tests/acceptance/features/apiComments/deleteComments.feature [hunk 4]
```diff
@@ -3,101 +3,99 @@ Feature: Comments
...
-    And user "Alice" has commented with content "<comment>" on file "/myFileToComment.txt"
+    Given user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/textfile0.txt"
+    And user "Alice" has commented with content "<comment>" on file "/textfile0.txt"
```
Comment (individual-it) [inline tests/acceptance/features/apiComments/deleteComments.feature:13]: why do we need to change those names? `myFileToComment` sounds like a good name

**Review by individual-it — CHANGES_REQUESTED** [review]
I don't understand why you do extra work and changing names of files/folders that work fine

**Review by Talank — COMMENTED** [review]
LGTM other than existing requests for changes

File: tests/acceptance/features/apiComments/deleteComments.feature (hunk 4 above)
Comment (Talank) [reply tests/acceptance/features/apiComments/deleteComments.feature:13]: the changes made the filename more simple but less descriptive

**Review by individual-it — APPROVED** [review]
still one place where you change the folder name for no reason and you also add `/` in some file actions :thinking: 
That just adds extra noise in the PR, but because the overall result seems to be OK I will approve it.

Please make sure the commit ids are bumped in ocis and cs3org/reva repos

File: tests/acceptance/features/apiComments/comments.feature [hunk 5]
```diff
@@ -36,8 +37,8 @@ Feature: Comments
     And the single response should contain a property "oc:comments-href" with value "%a_comment_url%"
 
   Scenario: Getting more info about comments using REPORT method
-    Given user "Alice" has uploaded file "filesForUpload/textfile.txt" to "myFileToComment.txt"
-    And user "Alice" has commented with content "My first comment" on file "myFileToComment.txt"
+    Given user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/myFileToComment.txt"
+    And user "Alice" has commented with content "My first comment" on file "/myFileToComment.txt"
```
Comment (Talank) [inline tests/acceptance/features/apiComments/comments.feature:41]: Why? 🤔 Is it really necessary to add `/`?

Comment (kiranparajuli589) [reply tests/acceptance/features/apiComments/comments.feature:41]: just be uniform in this feature file with the placement of `/`

**Pull request merged**